dmutils/email/dm_mailchimp.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+"""Digital Marketplace MailChimp integration."""
+
+from mailchimp3 import MailChimp
+from requests.exceptions import RequestException
+from hashlib import md5
+import six
+
+
+class DMMailChimpClient(object):
+
+    def __init__(
+        self,
+        mailchimp_username,
+        mailchimp_api_key,
+        logger
+    ):
+        self.client = MailChimp(mailchimp_username, mailchimp_api_key)
+        self.logger = logger
+
+    @staticmethod
+    def get_email_hash(email_address):
+        email_address = six.text_type(email_address.lower()).encode('utf-8')
+        return md5(email_address).hexdigest()
```

Comment:
md5 isn't very secure, we use sha256 here: https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/email/helpers.py#L10